cmd/pj-rehearse/main.go

```diff
@@ -87,34 +87,29 @@ an infrastructure issue.`
 
 pj-rehearse rehearsed jobs and at least one of them failed. This means that
 job would fail when executed against the current HEAD of the target branch.`
+	failedSetupOutput = `[ERROR] pj-rehearse: setup failure
+
+pj-rehearse failed to finish all setup necessary to perform job rehearsals.
+This is either a pj-rehearse bug or an infrastructure failure.`
 )
 
-func gracefulExit(suppressFailures bool, message string) {
+func gracefulExit(suppressFailures bool, message string) int {
 	if message != "" {
 		fmt.Fprintln(os.Stderr, message)
 	}
 
 	if suppressFailures {
-		os.Exit(0)
+		return 0
 	}
-	os.Exit(1)
+	return 1
 }
 
-func gracefulExitWithConfigMapCleanup(suppressFailures bool, message string, cmManager *config.TemplateCMManager) {
```

Comment:
Wait did we just nuke this?

Reply:
Yeah, it's no longer needed, this way you can `defer` the cleanup (https://github.com/openshift/ci-operator-prowgen/pull/110/files#diff-06579b6b828bbff5df82617adb17e7bfR196)